deployment/terraform/agent.go

```diff
@@ -225,7 +242,7 @@ func (t *Terraform) initLoadtest(extAgent *ssh.ExtAgent, initData bool) error {
 	if err != nil {
 		return err
 	}
-	dstPath := "/home/ubuntu/mattermost-load-test-ng/config/config.json"
+	dstPath := fmt.Sprintf("/home/%s/mattermost-load-test-ng/config/config.json", t.Config().AWSAMIUser)
```

Comment:
Same nit for these dstPath. Maybe it makes sense to get a common function `fromHomeDir` or something similar.

Reply:
https://github.com/mattermost/mattermost-load-test-ng/pull/919/commits/b794119eabfda6923ae5b592c40eeb61be380b10